source/ios/tools/AdaptiveCards.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |spec|
 
   spec.summary          = 'Adaptive Cards are a new way for developers to exchange card content in a common and consistent way'
   
-  spec.source       = { :git => 'https://github.com/microsoft/AdaptiveCards.git', :tag => 'ios-v2.9.3' }
+  spec.source       = { :git => 'https://github.com/microsoft/AdaptiveCards-Mobile.git', :tag => 'iOS/adaptivecards-ios@2.9.3' }
```

Comment:
do you need iOS/adaptivecards-ios@? if you are keep using the same tag, that's fine, if may be good to find a good tag to stick with.

Reply:
Yes, I changed that to follow android guideline for tags. This will help to maintain tags in folder structure for each platform.